.github/actions/setup-flexpret/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Install FlexPRET and dependencies (Linux only)
+description: Install FlexPRET and dependencies (Linux only)
+runs:
+  using: "composite"
+  steps:
+    - name: Setup
+      run: |
+        # Clone the FlexPRET repository
+        git clone --recurse-submodules https://github.com/pretis/flexpret
+
+        # This rest is copied directly from FlexPRET's `azure-pipelines.yml`
+
+        # Ubuntu 20.04 only has Verilator 4.028 but we neeed a more modern version
+        # so we do not use 'sudo apt-get install -y -qq verilator' here.
+        wget -q https://github.com/sifive/verilator/releases/download/4.036-0sifive2/verilator_4.036-0sifive2_amd64.deb -O verilator.deb
+        sudo dpkg -i verilator.deb
+
+        # Install riscv compiler
+        wget -q https://github.com/xpack-dev-tools/riscv-none-embed-gcc-xpack/releases/download/v8.3.0-2.3/xpack-riscv-none-embed-gcc-8.3.0-2.3-linux-x64.tar.gz -O riscv.tgz
```

Comment:
We can instead directly fetch the correct compiler, see this workflow: https://github.com/lf-lang/interpret/blob/main/.github/workflows/ci.yml

Reply:
Fixed :) Thanks for the tip!